mmdet/models/seg_heads/base_semantic_head.py

```diff
@@ -0,0 +1,86 @@
+from abc import ABCMeta, abstractmethod
+
+import torch.nn.functional as F
+from mmcv.runner import BaseModule, force_fp32
+
+from ..builder import build_loss
+from ..utils import interpolate_as
+
+
+class BaseSemanticHead(BaseModule, metaclass=ABCMeta):
+    """Base module of Semantic Head.
+
+    Args:
+        num_classes (int): the number of classes.
+        init_cfg (dict): the initialization config.
+        loss_seg (dict): the loss of the semantic head.
+    """
+
+    def __init__(self,
+                 num_classes,
+                 init_cfg=None,
+                 loss_seg=dict(
+                     type='CrossEntropyLoss', ignore_index=-1,
+                     loss_weight=1.0)):
+        super(BaseSemanticHead, self).__init__(init_cfg)
+        self.loss_seg = build_loss(loss_seg)
+        self.num_classes = num_classes
+
+    @force_fp32(apply_to=('logits', ))
+    def loss(self, logits, gt_semantic_seg, label_bias=1):
```

Comment:
Seems we seldom use  `logits` ， we use ` mask_pred` in all case

Reply:
Here use `seg_preds` to distinguish from `mask_preds` in mask heads.